builtin/providers/aws/resource_aws_iam_role_policy_attachment_test.go

```diff
@@ -7,30 +7,35 @@ import (
 
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/service/iam"
+	"github.com/hashicorp/terraform/helper/acctest"
 	"github.com/hashicorp/terraform/helper/resource"
 	"github.com/hashicorp/terraform/terraform"
 )
 
 func TestAccAWSRolePolicyAttachment_basic(t *testing.T) {
 	var out iam.ListAttachedRolePoliciesOutput
+	rInt := acctest.RandInt()
+	testPolicy := fmt.Sprintf("test-policy-%d", rInt)
+	testPolicy2 := fmt.Sprintf("test-policy2-%d", rInt)
+	testPolicy3 := fmt.Sprintf("test-policy3-%d", rInt)
```

Comment:
Would you mind changing this to something like `tf-acc-test-%d` so it's more obvious - in case of a leak - that those resources come from an acceptance test?